dexter/src/main/java/com/karumi/dexter/Dexter.java

```diff
@@ -53,8 +138,10 @@
    *
    * @param activity context used by Dexter. Remember to invoke {@link com.karumi.dexter.Dexter}
    * stop method to avoid memory leaks.
+   * @deprecated There is no need to initialize Dexter anymore. Use `Dexter.withActivity()` to
+   * perform any permission check.
    */
-  public static void initialize(Activity activity) {
+  @Deprecated public static void initialize(Activity activity) {
```

Comment:
We don't need to deprecate this method because we didn't releasy a version with this new code :)